x-pack/elastic-agent/pkg/agent/cmd/run.go

```diff
@@ -135,21 +142,33 @@ func run(streams *cli.IOStreams, override cfgOverrider) error {
 
 	statusCtrl := status.NewController(logger)
 
+	// TODO: How do we get the version?
```

Comment:
`release.Version()` as you have below should be the version of the agent (unless you wanted a different version number?)

Reply:
Nope, that sounds perfect